vpr/src/base/vpr_api.cpp

```diff
@@ -1207,9 +1208,9 @@ void vpr_analysis(t_vpr_setup& vpr_setup, const t_arch& Arch, const RouteStatus&
 
         //Timing stats
         VTR_LOG("\n");
-        generate_hold_timing_stats(/*prefix=*/"", *timing_info,
+        generate_hold_timing_stats(/*prefix=*/vpr_setup.AnalysisOpts.timing_report_prefix, *timing_info,
                                    *analysis_delay_calc, vpr_setup.AnalysisOpts);
-        generate_setup_timing_stats(/*prefix=*/"", *timing_info,
+        generate_setup_timing_stats(/*prefix=*/vpr_setup.AnalysisOpts.timing_report_prefix, *timing_info,
```

Comment:
This should also be removed.

The prefix is used when printing out information to stdout/vpr log, since this is called in multiple places (e.g. post-place, post-route). It should not have anything set here.